src/modules/forking/components/forking-notification/forking-notification.jsx

```diff
@@ -90,6 +91,7 @@ ForkingNotification.propTypes = {
   forkEndTime: PropTypes.string.isRequired,
   currentTime: PropTypes.number.isRequired,
   doesUserHaveRep: PropTypes.bool.isRequired,
+  marginLeft: PropTypes.string.isRequired,
 }
```

Comment:
Getting this error, making marginLeft a number instead of string, pushing commit
![screen shot 2018-06-04 at 7 56 21 pm](https://user-images.githubusercontent.com/3970376/40948999-6152e77c-6831-11e8-93d1-ce29a42e6453.png)

Reply:
works for me 👍